### PR TITLE
Restructure of Virology Section

### LINKED ISSUE #615: Intro TODO "...issues regarding the SARS-CoV-2 evolutionary pattern."
**Is your feature request related to a problem? Please describe.**

I read this sentence and couldn't figure out what the issues were, exactly.

**Describe the solution you'd like**

Maybe someone who understands the sentence could take a stab at re-wording it? Alternatively we could probably remove that clause.

**Describe alternatives you've considered**

Removing the clause in question + the TODO.

**Additional context**

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This is a fluidity edit of the current virology section. 
Going back on my own opinions of two weeks ago, I actually think this text should move to the beginning of the pathogenesis section.
However, because I made so many changes to the text, I will submit that move as a separate PR following merge so that the changes are clear here.

@Sergey-Knyazev I am tagging you because I made some edits to your text and wanted to run it by you to make sure your message was preserved.

### Related issues

closes #615 
related to #612 
Does not address #614 (to follow)

### Suggested reviewers (optional)

@cgreene @Sergey-Knyazev 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/02.introduction.md [hunk 1]
```diff
@@ -61,22 +61,29 @@ Fortunately, the basic genome structure of coronaviruses is highly conserved, an
...
+In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were largely confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].
+
+Since the turn of the millenium, two HCoVs of global concern have also been identified.
+At the time that SARS-CoV emerged in the early 2000s, no HCoV had been identified in almost 40 years [@doi:10.1038/nrmicro.2016.81].
+The first caseof SARS was reported in November 2002 in the Guangdong Province of China, and over the following month, the disease spread more widely within China and then into several countries across multiple continents [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81].
+Unlike previously identified HCoV, SARS was severe, with an estimated death rate of 9.5% [@doi:10.1093/ajcp/aqaa029].
```
Comment (Sergey-Knyazev) [inline content/02.introduction.md:78]: may we emphasize "much severe"?

File: content/02.introduction.md [hunk 2]
```diff
@@ -104,9 +105,54 @@ It also includes cell-surface receptors that can recognize the molecular pattern
...
+Another strategy that SARS-CoV utilizes to increase its virulence is the suppression of host gene expression by nonstructural protein 1 (Nsp1).
+Nsp1 accomplishes this by stalling mRNA translation and inducing endonucleolytic cleavage and mRNA degradation [@doi:10.1126/science.abc8665].
+These evasion mechanisms, in turn, can lead to systemic infection.
+
+SARS-CoV also evades the immune response by interfering with type I IFN induction signaling, which is a mechanism that leads to cellular resistance to viral infections.
+SARS-CoV employs methods such as ubiquitination and degradation of RNA sensor adaptor molecules MAVS and TRAF3/6 [@doi:10.12932/AP-200220-0772].
+MERS-CoV also utilizes these mechanisms, as well as histone modification.
+Also, MERS-CoV downregulates antigen presentation via MHC class I and MHC class II, which leads to a reduction in T cell activation [@doi:10.12932/AP-200220-0772].
+
+Based on the aforementioned mechanisms for both SARS-CoV and MERS-CoV, SARS-CoV-2 most likely utilizes similar mechanisms for immune evasion.
```
Comment (Sergey-Knyazev) [inline content/02.introduction.md]: I would put a citation here

File: content/02.introduction.md [hunk 3]
```diff
@@ -61,23 +61,30 @@ Fortunately, the basic genome structure of coronaviruses is highly conserved, an
 
 ### Coronaviruses and Humans Hosts
 
-Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals [@doi:10/ckfn8b]..
-The first coronaviruses that infect humans (hCoV) were identified in the 1960s [@doi:10.1136/bmj.1.5448.1467; @doi:10.3181/00379727-121-30734].
+Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals [@doi:10/ckfn8b].
+Most coronaviruses show little to no transmission observed in humans.
```
Comment (rando2) [inline content/02.introduction.md:65]:
```suggestion
Most coronaviruses show little to no transmission in humans.
However, today it is thought that approximately one-third of common cold infections are caused by four HCoV (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
```

File: content/02.introduction.md [hunk 4]
```diff
@@ -61,22 +61,29 @@ Fortunately, the basic genome structure of coronaviruses is highly conserved, an
...
+Two additional HCoV were subsequently identified [@doi:10.1038/nm1024; @doi:10.1128/JVI.79.2.884-895.2005].
+In 2003, HCoV-NL63 [@doi:10.1038/nm1024] was identified in a 7-month-old infant and then identified in clinical specimens collected from seven additional patients, five of whom were infants (younger than 1 year) and the remainder of whom were adults.
+CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and identified in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
+These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612].
+Approximately one-third of common cold infections are caused by four HCoV (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that circulate in the global human population [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/02.introduction.md:72]: Good idea -- moved it above and defined all of the species names!
```suggestion
```

File: content/02.introduction.md [hunk 5]
```diff
@@ -61,23 +61,30 @@ Fortunately, the basic genome structure of coronaviruses is highly conserved, an
 
 ### Coronaviruses and Humans Hosts
 
-Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals [@doi:10/ckfn8b]..
-The first coronaviruses that infect humans (hCoV) were identified in the 1960s [@doi:10.1136/bmj.1.5448.1467; @doi:10.3181/00379727-121-30734].
+Coronaviruses have long been known to infect animals and have been the subject of veterinary medical investigations and vaccine development efforts due to their effect on the health of companion and agricultural animals [@doi:10/ckfn8b].
+Most coronaviruses show little to no transmission observed in humans.
+The first coronaviruses that infect humans (HCoV) were identified in the 1960s [@doi:10.1136/bmj.1.5448.1467; @doi:10.3181/00379727-121-30734].
```
Comment (rando2) [inline content/02.introduction.md:66]:
```suggestion
The first coronavirus species infecting humans (human coronavirus or HCoV) were identified in the 1960s: HCoV-229E in 1965 [@doi:10.3181/00379727-121-30734] and HCoV-OC43 in 1967 [@doi:10.1073/pnas.57.4.933].
```

File: content/02.introduction.md [hunk 6]
```diff
@@ -61,22 +61,29 @@ Fortunately, the basic genome structure of coronaviruses is highly conserved, an
...
+CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and identified in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
+These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612].
+Approximately one-third of common cold infections are caused by four HCoV (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that circulate in the global human population [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
+In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were largely confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/02.introduction.md:73]:
```suggestion
In addition to these relatively mild HCoV, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV or SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were largely confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].
```

File: content/02.introduction.md [hunk 7]
```diff
@@ -61,23 +61,30 @@ Fortunately, the basic genome structure of coronaviruses is highly conserved, an
...
+
+Since the turn of the millennium, two HCoVs of global concern have also been identified.
+At the time that SARS-CoV emerged in the early 2000s, no HCoV had been identified in almost 40 years [@doi:10.1038/nrmicro.2016.81].
+The first case of SARS was reported in November 2002 in the Guangdong Province of China, and over the following month, the disease spread more widely within China and then into several countries across multiple continents [@doi:10.1093/ajcp/aqaa029; @doi:10.1038/nrmicro.2016.81].
+Unlike previously identified HCoV, SARS was severe, with an estimated death rate of 9.5% [@doi:10.1093/ajcp/aqaa029].
```
Comment (rando2) [inline content/02.introduction.md:78]:
```suggestion
Unlike previously identified HCoV, SARS was much more severe, with an estimated death rate of 9.5% [@doi:10.1093/ajcp/aqaa029].
```

File: content/02.introduction.md [hunk 8]
```diff
@@ -61,23 +61,30 @@ Fortunately, the basic genome structure of coronaviruses is highly conserved, an
...
+CoV-HKU1 was identified in samples collected from a 71-year-old pneumonia patient in 2004 and identified in samples collected from a second adult patient [@doi:10.1128/JVI.79.2.884-895.2005].
+These viruses are associated with respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612].
+Approximately one-third of common cold infections are caused by four HCoV (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that circulate in the global human population [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
+In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were largely confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81; @doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].
+
+Since the turn of the millennium, two HCoVs of global concern have also been identified.
```
Comment (rando2) [inline content/02.introduction.md:75]:
```suggestion
```

Per <https://github.com/greenelab/covid19-review/pull/628#discussion_r480630255>

File: content/02.introduction.md (hunk 2 above)
Comment (rando2) [reply content/02.introduction.md]: I can't seem to edit this here, so linking it into #349

**Pull request merged**